test/e2e/tests/confirmations/helpers.ts

```diff
@@ -36,7 +40,7 @@ export function withRedesignConfirmationFixtures(
         .build(),
       ganacheOptions: defaultGanacheOptions,
       title,
-      testSpecificMock: mockSegment,
+      testSpecificMock: testSpecificMockOverride || mockSegment,
```

Comment:
hi there :wave:  the custom fixture function seems a bit confusing, but it looks like the test is failing because you are not passing the smartContract, meaning this won't be deployed. Then, when it tries to get the contract address, in the `openDAppWithContract` that is undefined:

`TypeError: Cannot read properties of undefined (reading 'getContractAddress')`